db/migrate/20220917131536_add_nudge_email_to_requests.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddNudgeEmailToRequests < ActiveRecord::Migration[6.0]
+  def change
+    add_column :requests, :nudge_email_opt_out, :bool, default: false
+    add_column :requests, :nudge_email_opt_out_date, :timestamp
```

Comment:
I have tended towards using datetime rather than timestam as a data type, but both should work and appear the same. Saving on space for timestamp, but don't think it'll see beyond 2030-something.

Reply:
That field isn't working properly yet anyway and I didn't remove it. (bad Huddo)